geopyspark/geotrellis/layer.py

```diff
@@ -255,12 +255,63 @@ def to_tiled_layer(self, extent=None, layout=None, crs=None, tile_size=256,
             ``extent`` and ``layout`` must both be defined if they are to be used.
 
         Returns:
-            :class:`~geopyspark.geotrellis.rdd.TiledRasterLayer`
+            :class:`~geopyspark.geotrellis.rdd.RasterLayer`
         """
 
         return self.tile_to_layout(self.collect_metadata(extent, layout, crs, tile_size),
                                    resample_method)
 
+    def bands(self, band):
```

Comment:
It seemed initially that this should have been called `subsetBands` to mirror GT method name. But for the purpose of the python API `bands` seems plenty clear.